l10n_eu_nace/migrations/12.0.2.0.0/pre-migration.py

```diff
@@ -6,7 +6,7 @@ def migrate(cr, version):
     cr.execute(
         """
         UPDATE ir_model_data
-        SET name=concat('old_', name), noupdate=1
+        SET name=concat('old_', name), noupdate=TRUE
```

Comment:
Is this really a bug? Do you have migration logs on the subject?

I mean, I'm :+1: with this change anyways, but I'm curious about this for future migrations.

Reply:
PgSQL 12 was insulting me on this that's why I proposed this patch.